scripts/check_docs

```diff
@@ -6,8 +6,10 @@ readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
 # shellcheck source=scripts/common
 source "$SCRIPTS_DIR/common"
 
+DOCS_OUT="$(cargo doc --document-private-items --no-deps 2>&1)"
```

Comment:
Does this print the logs already, or do we need to print the variable in the if body?

Reply:
Yes. Because of `set -o xtrace` in `common`, which is included here, everything that is executed is printed. I could echo `DOCS_OUT` in the `if`, but then we'd get two copies of the logs.